### PR TITLE
feat: cssVar could be disabled by set to false

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |    -       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 1f6b552</samp>

This pull request implements a CSS variable-based theme animation feature for the `ant-design` library. It adds a `cssVar` prop to the `GlobalLayout` component, modifies the `useTheme` and `useThemeAnimation` hooks, and updates the `Button` component's CSS variable registration.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 1f6b552</samp>

* Enable CSS variable-based theme animation for `GlobalLayout` component by adding `cssVar` prop ([link](https://github.com/ant-design/ant-design/pull/45987/files?diff=unified&w=0#diff-5d7076c9ed6d07433cbe0e9ec9d4940ec94b53422e0475998ac2ce2e9caa3b46R180))
* Add `time` variable to measure theme transition duration in `useThemeAnimation` hook ([link](https://github.com/ant-design/ant-design/pull/45987/files?diff=unified&w=0#diff-941fcbacf1a38746130e95a5c46349e59999f00997963e17e027e79e2840a2c0R71))
* Log theme transition duration in console after animation ends ([link](https://github.com/ant-design/ant-design/pull/45987/files?diff=unified&w=0#diff-941fcbacf1a38746130e95a5c46349e59999f00997963e17e027e79e2840a2c0R102))
* Modify `mergedCssVar` object in `useTheme` hook to use `ant` prefix and nullish coalescing operator ([link](https://github.com/ant-design/ant-design/pull/45987/files?diff=unified&w=0#diff-85ff925f8aa07cbb863b96b5bffc960720ad00516700939d2aa40cde582d41f3L53-R54))
* Specify unitless CSS properties for `Button` component in `genCSSVarRegister` function call ([link](https://github.com/ant-design/ant-design/pull/45987/files?diff=unified&w=0#diff-da465e13e5a6616903efcbebef2ee41cf92990ed7325110b8fde64d7367a7836L4-R8))

### CODE REVIEW

**Comment by yoyo837** [comment]
argos的slider是正常的不?

**Review by yoyo837 — APPROVED** [review]

File: .dumi/theme/layouts/GlobalLayout.tsx [hunk 1]
```diff
@@ -177,6 +177,7 @@ const GlobalLayout: React.FC = () => {
               token: {
                 motion: !theme.includes('motion-off'),
               },
+              cssVar: true,
```
Comment (kiner-tang) [inline .dumi/theme/layouts/GlobalLayout.tsx:180]: 这里固定设置为 `true` 要怎么禁用？

Comment (MadCcc) [reply .dumi/theme/layouts/GlobalLayout.tsx:180]: 可以在内部的 ConfigProvider 里强制设为 false

Comment (yoyo837) [reply .dumi/theme/layouts/GlobalLayout.tsx:180]: 这个是站点的, 没关系吧?

**Review by li-jia-nan — APPROVED** [review]

File: .dumi/theme/layouts/GlobalLayout.tsx (hunk 1 above)
Comment (kiner-tang) [reply .dumi/theme/layouts/GlobalLayout.tsx:180]: 原来如此

**Comment by li-jia-nan** [comment]
> argos的slider是正常的不? 

正常的

**Review by kiner-tang — APPROVED** [review]

**Pull request merged**